packages/trpc/server/routers/loggedInViewer/integrations.handler.ts

```diff
@@ -33,8 +34,16 @@ type TeamQuery = Prisma.TeamGetPayload<{
 
 export const integrationsHandler = async ({ ctx, input }: IntegrationsOptions) => {
   const { user } = ctx;
-  const { variant, exclude, onlyInstalled, includeTeamInstalledApps, extendsFeature, teamId } = input;
-  let { credentials } = user;
+  const {
+    variant,
+    exclude,
+    onlyInstalled,
+    includeTeamInstalledApps,
+    extendsFeature,
+    teamId,
+    sortByMostPopular,
+  } = input;
+  const { credentials } = user;
```

Comment:
```suggestion
  let { credentials } = user;
```
causes type error, as we assign a new value to `credentials` later on

Reply:
Ok now looking back at it was a mistake from resolving conflicts. The ide showed the error but i couldn't find why, but it all makes sense now. Thanks for that @CarinaWolli !!